From: Philip Withnall <withnall@endlessm.com>
Date: Wed, 29 Aug 2018 12:02:17 +0100
Subject: build: Stop defining STDC_HEADERS
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

This was previously defined by the AC_HEADER_STDC macro in configure.ac,
but we don’t use that any more. Nothing in GLib depends on this macro,
and neither does anything in my /usr/include which we might care about.

The autoconf documentation for AC_HEADER_STDC says it’s deprecated:
https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Particular-Headers.html#index-AC_005fHEADER_005fSTDC-621

Signed-off-by: Philip Withnall <withnall@endlessm.com>

Bug: https://gitlab.gnome.org/GNOME/glib/issues/1313
Origin: upstream, 2.59.0, commit:e42d5dafb74a6f7f451a1b34cfede741edbf8f24
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index d583603..ee5deb7 100644
--- a/meson.build
+++ b/meson.build
@@ -1884,7 +1884,6 @@ have_sh = find_program('sh', required : false).found() # For glib-gettextize
 
 # FIXME: defines in config.h that are not actually used anywhere
 # (we add them for now to minimise the diff)
-glib_conf.set('STDC_HEADERS', 1)
 glib_conf.set('SIZEOF___INT64', 8)
 
 # FIXME: How to detect Solaris? https://github.com/mesonbuild/meson/issues/1578
